### PR TITLE
Development

### PR DESCRIPTION
**Pull request by karthykarthick**
Hello TSE's,

we complete our Fourth Project. If there any error feel free to mention it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
...
+
+![screenshot](./src/webpage_screenshot.png)
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+- [Live Demo Link](https://raw.githack.com/Ahmed-Benj/apple-website-clone/tree/development)
```
Comment (elmejdki) [inline README.md]: please fix the live demo link it's not working

File: README.md [hunk 2]
```diff
@@ -1 +1,49 @@
-# apple-website-clone
\ No newline at end of file
+# apple-website-clone
+
+# PROJECT: PROJECT: BUILDING WITH BACKGROUNDS AND GRADIENTS (Microverse Collaborative Project)
```
Comment (elmejdki) [inline README.md]: Fix the title

File: index.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            <br />
+            <h2 id="sub-txt">Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.</h2>
+            <h3 class="sub-heading">Watch the “Your Verse” film and explore the stories</h3>
+        </div>
+      </div>
+      <div class="image1"></div>
+      <div class="image2"></div>
+      <div class="image3"></div>
+      <div class="image4"></div>
+
```
Comment (elmejdki) [inline index.html]: these images should be links please wrap them inside a link tag or change the div into a link tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    />
+    <title>Apple</title>
+
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header>
+      
+      <div class="image0">
```
Comment (elmejdki) [inline index.html]: The CSS background should cover the whole page irrespective of the screen size. 
image

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Status: require changes :recycle: 
## Great job guys :clap: , your clone looks super cool :open_mouth:, but you need to make some changes in order to get it approved.

I have left for you some comments bellow please fix them before you request another code review.

## Happy Coding
_If you have any questions don't hesitate to contact me on slack_

File: README.md (hunk 1 above)
Comment (Ahmed-Benj) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (Ahmed-Benj) [reply README.md]: done

File: index.html (hunk 3 above)
Comment (Ahmed-Benj) [reply index.html]: doing so, wrapping them inside a `<a>` or `<link>` tags make them disappear from the page

**Comment by Ahmed-Benj** [comment]
Hello @elmejdki ,

Thank you for your comments and detailed review, we are working on the fixes.

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @karthykarthick  and @Ahmed-Benj! 👋 
You have done a great job implementing the changes requested by the previous TSE! 👍 You have also done a wonderful job with your project so far! It looks really good! 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [ ] The background should cover the whole page irrespective of screen size. This was pointed out to you by the previous TSE and unfortunately, you have not yet fixed this. Please see the screenshots below and compare your clone with the original.
Your clone 👇 
Screen Shot 2020-06-30 at 7 57 51 PM
The original page 👇 
Screen Shot 2020-06-30 at 7 57 57 PM

- [x] The gradient of the navbar is not quite implemented correctly. Yours appears to gradually lighten from left to right while the original goes from bottom to top. Please check [this article](https://www.w3schools.com/css/css3_gradients.asp) for more information on how to implement the gradient. 
Your clone 👇 
Screen Shot 2020-06-30 at 8 06 27 PM
The original page 👇 
Screen Shot 2020-06-30 at 8 06 49 PM
_Hint: The default for the linear gradient is from top to bottom so if you remove the 90 deg and play around with the order of the colors you should be able to get it looking like the original in no time_ 😄 


****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by Ahmed-Benj** [comment]
done the navbar color with : `    background-image: linear-gradient(#888888,#5F5F5F,#6D6D6D);`

**Comment by Ahmed-Benj** [comment]
Hello @BrittanyBlake ,

Thank you for your review, I want to mention that the woriginal webpage is not shown on fullscreen on our display, like this : 
image

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🛑

## Hi team 👋. Thanks for making the changes. But there are some changes that you missed. Let's see:

- The background color of the navbar is not right. Right now the dark part is concentrated where I have marked with red.

#### Clone
Annotation 2020-07-01 005048

But the shades must be on top of each other and evenly distributed throughout its width.

#### Original
Annotation 2020-07-01 005019

Even the picture you shared in the comments, this is the case. Please fix this.

- These four images must be links.
image

Apply these changes and request another review.

Happy coding 👨‍💻

**Review by tirthajyoti-ghosh — APPROVED** [review]
# Status: Approved ✅

## Congratulations team 🎉! It's a 👍 from me. All requirements are met. Go ahead and merge it.

### Onto the next project 🚀!

Happy coding 👨‍💻

**Pull request merged**